src/components/EditGrower.js

```diff
@@ -126,6 +141,16 @@ const EditGrower = (props) => {
             }
             onSelectChange={handleChange}
           />
+          <TextField
+            className={classes.textInput}
+            label="Image Custom URl"
+            error={!isImgURL(customUrl)}
```

Comment:
This shows as an error when the user opens the dialog before they've done anything – not a great user experience.
Instead, only check when they blur the input, and display a note explaining the error ("Invalid image URL").

Reply:
i solved thise with a state
![image](https://user-images.githubusercontent.com/100769556/222033503-b4d2b30d-a85b-48e5-8426-55ce3517e8fb.png)
![image](https://user-images.githubusercontent.com/100769556/222033540-64f8ceac-07f3-4e2b-9b1a-7b7f8ebf22f2.png)
![image](https://user-images.githubusercontent.com/100769556/222033822-0c540982-5980-4b65-bb7e-8a2e93906a6c.png)